src/internal/packager/helm/zarf.go

```diff
@@ -117,14 +117,14 @@ func (h *Helm) UpdateZarfAgentValues() error {
 	for _, lsRelease := range releases {
```

Comment:
can this just be called `release` and not `lsRelease`?

Reply:
yes